src/main/twirl/uk/gov/hmrc/play/views/helpers/singleCheckbox.scala.html

```diff
@@ -19,13 +19,13 @@
 @import views.html.helper._
 
 @elements = @{ new FieldElements(field.id, field, null, args.toMap, messages) }
-@value = @{ field.value match { case Some(x) => x case None => "false" case x => x }}
-<div class="form-field-single @if(elements.hasErrors) {error}">
+@value = @{ field.value match { case Some(x) => x case None => "false" case x => x } }
```

Comment:
This feels odd, its matching an `Option` (`Some` or `None`) or a string. Where as `field.value` is an `Option` https://playframework.com/documentation/2.0/api/scala/play/api/data/Field.html